divi/src/masternode.h

```diff
@@ -250,6 +251,10 @@ class CMasternode
 
 class CMasternodeBroadcast : public CMasternode
 {
+private:
+
+    friend class CMasternodeBroadcastFactory;
+
```

Comment:
Not sure why this friend declaration is needed here. Other than that this PR is very straightforward

Reply:
Indeed.  In a previous version of the code, `GetCollateralConfirmationBlock` was `private`, and that is why the `friend` was needed.  Not any more now, so this can go.